src/sentry/integrations/jira_server/webhooks.py

```diff
@@ -51,16 +58,12 @@ def dispatch(self, request: Request, *args, **kwargs) -> Response:
         return super().dispatch(request, *args, **kwargs)
 
     def post(self, request: Request, token, *args, **kwargs) -> Response:
-        extra = {}
-        try:
-            integration = get_integration_from_token(token)
-            extra["integration_id"] = integration.id
-        except ValueError as err:
```

Comment:
Removing the except block will cause the error to be unhandled. Do we have a top level catch? Otherwise, it would affect the top level process that handles the webhooks.